test/requirements.txt

```diff
@@ -1,5 +1,4 @@
-apache-airflow==1.10.11
-apache-beam==2.23.0
-six==1.14.0
-dill==0.3.0
+apache-airflow>=1.10
 nose==1.3.7
+mock>=2.0
```

Comment:
Same case as with airflow

Reply:
I've changed to pinned version for all test dependencies except airflow.